src/routes/write-exercise/__tests__/WriteExerciseScreen.spec.tsx

```diff
@@ -58,7 +59,9 @@ describe('WriteExerciseScreen', () => {
     // @ts-expect-error because typescript does not know FiberNode
     const getUri = (image: ReactTestInstance) => image._fiber.stateNode.props.source.uri
 
-    const { getByRole, getByText } = render(<WriteExerciseScreen route={route} navigation={navigation} />)
+    const { getByRole, getByText } = render(<WriteExerciseScreen route={route} navigation={navigation} />, {
+      wrapper: wrapWithTheme
+    })
     const image = await getByRole('image')
```

Comment:
running the test fails with error message `Error: No instances found with accessibilityRole "image"`.
Any ideas on how to fix this? @SteffiStuffel @Taggotty